code/modules/events/ghost_role/pirates.dm

```diff
@@ -97,9 +97,31 @@
 		if(PIRATES_IMPERIAL_ENCLAVE)
 			ship_name = pick(strings(PIRATE_NAMES_FILE, "imperial_names"))
 			ship_template = /datum/map_template/shuttle/pirate/imperial_enclave
+			payoff = 20000
+			var/number = rand(1,999)
+			///Station name one is the most important pick and is pretty much the station's main argument against getting fined, thus it better be mostly always right.
+			var/station_designation = pick_weight(list("Nanotrasen Research Station" = 70,
+												"Nanotrasen Refueling Outpost" = 5,
+												"Interdyne Pharmaceuticals Chemical Factory" = 5,
+												"Free Teshari League Engineering Station" = 5,
+												"Agurkrral Military Base" = 5,
+												"Sol Federation Interceptor" = 5,
```

Comment:
Threat.content has the text 'your station', Interceptor is a kind of ship :b While the report's meant to be occasionally wrong, it shouldn't be contradictory

Reply:
I assume that was fixed.